src/tiles/list/tile-list.ts

```diff
@@ -0,0 +1,326 @@
+              this.date,
+              this.formatSize
+            )}
+          </div>
+          <div id="views-line">
+            ${this.views()} ${this.rating()} ${this.reviews()}
+          </div>
+          ${this.topics()} ${this.description()}
+        </div>
+      </div>
+    `;
+  }
+
+  private img() {
+    if (!this.model?.identifier) {
+      return html``;
+    }
+    return html` <img
+      src="${this.baseNavigationUrl}/services/img/${this.model.identifier}"
+      alt="${this.model.identifier}"
+      class=${this.model?.mediatype}
+    />`;
+  }
+
+  private iconLeft() {
+    if (this.classSize !== 'desktop') {
+      return html``;
+    }
+    return html`
+      <div id="icon-left">
+        <mediatype-icon .mediatype=${this.model?.mediatype} .showText=${true}>
+        </mediatype-icon>
+      </div>
+    `;
+  }
+
+  private creator() {
+    if (!this.model?.creator) {
+      return html``;
+    }
+    return html`
+      <div id="creator">
+        <span class="label">By: </span>
+        ${DOMPurify.sanitize(this.model?.creator ?? '')}
+      </div>
+    `;
+  }
+
+  private itemLine() {
+    const source = this.source();
+    if (!source) {
+      return html``;
+    }
+    return html` <div id="item-line">${source}</div> `;
+  }
+
+  private views() {
+    return html`
+      <div id="views">
+        <span class="label">Views: </span>
+        ${formatCount(this.model?.viewCount ?? 0, this.formatSize)}
+      </div>
+    `;
+  }
+
+  private rating() {
+    if (!this.model?.averageRating) {
+      return html``;
+    }
+    return html`
+      <div id="reviews">
+        <span class="label">Avg Rating: </span>
+        ${this.model?.averageRating}
+      </div>
+    `;
+  }
+
+  private reviews() {
+    if (!this.model?.commentCount) {
+      return html``;
+    }
+    return html`
+      <div id="reviews">
+        <span class="label">Reviews: </span>
+        ${this.model?.commentCount}
+      </div>
+    `;
+  }
+
+  private description() {
+    if (!this.model?.description) {
+      return html``;
+    }
+    const description = DOMPurify.sanitize(`${this.model?.description}`);
+    return html`<div id="description">${description}</div>`;
+  }
+
+  private topics() {
+    if (!this.model?.subject) {
+      return html``;
+    }
+    return html` <div id="topics">
+      <span class="label">Topics: </span>
+      ${DOMPurify.sanitize(this.model?.subject)}
+    </div>`;
+  }
+
+  private source() {
+    if (this.model?.source) {
+      return html`<div id="source">
+        <span class="label">Source: </span>${this.searchLink(
+          'source',
+          this.model.source
+        )}
+      </div>`;
+    }
+    return html``;
+  }
+
+  private searchLink(field: string, searchTerm: string) {
+    if (!field || !searchTerm) {
+      return html``;
+    }
+    const query = encodeURIComponent(`${field}:"${searchTerm}"`);
+    // eslint-disable-next-line lit/no-invalid-html
```

Comment:
Q: what invalid html was this detecting?

Reply:
Turns out is was the "" in query inside the "" for href=""
Fixed